sql/types/datetime.go

```diff
@@ -390,9 +404,9 @@ func (t datetimeType) String() string {
 	case sqltypes.Date:
 		return "date"
 	case sqltypes.Datetime:
-		return "datetime(6)"
+		return fmt.Sprintf("datetime(%d)", t.precision)
```

Comment:
If the precision is zero, then we should just display `datetime` and `timestamp`, as `datetime(0)` doesn't look great.

Reply:
Good call